bug_change_status_page.php

```diff
@@ -158,7 +159,7 @@
 
 <?php
 	$t_current_resolution = $t_bug->resolution;
-	$t_bug_is_open = $t_current_resolution < $t_resolved;
+	$t_bug_is_open = $t_current_resolution < $t_resolution_fixed;
```

Comment:
>  I vaguely remember discussing this before

Found it: https://www.mantisbt.org/bugs/view.php?id=15653#c37182
Hard to believe, I thought we discussed it maybe one year ago, but it's been more than three and a half years ...